README.md

```diff
@@ -41,6 +41,23 @@ const sql = SQL`
 db.query(sql) // execute query
 ```
 
+### Raw values
+
+If you need to pass a dynamic string to the query that MUST NOT be passed as query parameter, enclose the parameter in a object with the `__raw` property.
+
+```js
+const SQL = require('@nearform/sql')
+
+const db = connectDB() // your db instance
+
+const field = 'username'
+
+// generate SQL query
+const sql = SQL`SELECT ${{__raw: field}} FROM users`
+
+db.query(sql) // execute query
+```
```

Comment:
I'm somehow uncomfortable to support this feature at all. Whatever you are passing as `{__raw: field}` it's going to be __vulnerable to SQL injection__.